javalin/src/test/java/io/javalin/TestStaticFiles.kt

```diff
@@ -34,15 +34,17 @@ class TestStaticFiles {
     private val defaultStaticResourceApp: Javalin by lazy { Javalin.create { it.addStaticFiles("/public") } } // classpath
     private val externalStaticResourceApp: Javalin by lazy { Javalin.create { it.addStaticFiles("src/test/external/", Location.EXTERNAL) } }
     private val multiLocationStaticResourceApp: Javalin by lazy {
-        Javalin.create { servlet ->
-            servlet.addStaticFiles("src/test/external/", Location.EXTERNAL)
-            servlet.addStaticFiles("/public/immutable")
-            servlet.addStaticFiles("/public/protected")
-            servlet.addStaticFiles("/public/subdir")
+        Javalin.create {
+            it.aliasCheckForStaticFiles = ContextHandler.AliasCheck { path, resource -> !path.endsWith(".txt") }
+            it.addStaticFiles("src/test/external/", Location.EXTERNAL)
+            it.addStaticFiles("/public/immutable")
+            it.addStaticFiles("/public/protected")
+            it.addStaticFiles("/public/subdir")
         }
     }
     private val devLoggingApp: Javalin by lazy {
         Javalin.create {
+            it.aliasCheckForStaticFiles = ContextHandler.AliasCheck { path, resource -> !path.endsWith(".txt") }
```

Comment:
Same as above, no usage of this aliasCheck?

Reply:
Fixed!